### PR TITLE
Development

### PR DESCRIPTION
**Pull request by taiwo-sam**

### CODE REVIEW

File: spec/invertedSpec.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.',
+  },
+];
+
```
Comment (fortunee) [inline spec/invertedSpec.js]: Moving the mocked JSON data to a separate file and importing them into your test will make your code more modularized

File: spec/invertedSpec.js [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+        an: [1],
+        elf: [1],
+        falls: [0],
+        hole: [0],
+        into: [0],
+        man: [1],
+        of: [1],
+        rabbit: [0],
+        unusual: [1] }, 'doc.json']);
+    });
```
Comment (fortunee) [inline spec/invertedSpec.js]: You should consider having the expected result in a variable like this 
` const result = [{ a: [0],
        alice: [0],
        alliance: [1],
         an: [1],
         elf: [1],
         falls: [0],
         hole: [0],
         into: [0],
         man: [1],
     of: [1],
    rabbit: [0],
    unusual: [1] }, 'doc.json']; `
then pass the variable like this 
`.toEqual(result)`. 
This makes your code look clean and readable.

**Comment by michael-alade** [comment]
## Review
- Your method comments are not consistent in your inverted index file. Remove any `//` characters.
- The variable type is not consistent. It should be more like this 👇 
```js
/**
* Whatever method name
* @desc This description of the method or function
* @param {String} testVariable - whatever it does
* @param {Object} anObject - whatever
* @return {void}
**/
```

Notice that `@param` value starts with an uppercase character.

- In my own opinion, I would not use javascript classes, if you want to gain a clear sense of what I mean looking into this https://www.quora.com/Are-ES6-classes-bad-for-JavaScript. You can functions.

File: spec/invertedSpec.js (hunk 2 above)
Comment (taiwo-sam) [reply spec/invertedSpec.js]: Okay. Thanks!

File: spec/invertedSpec.js (hunk 1 above)
Comment (taiwo-sam) [reply spec/invertedSpec.js]: Okay. I get it. Will make it that way. Thanks!

File: .eslintrc.json [hunk 3]
```diff
@@ -1,6 +1,12 @@
 {
```
Comment (raphaelakpan) [inline .eslintrc.json:1]: Your `.eslintrc` file does not seem to have the minimum rules that is required to properly format your code. You should consider getting the required configurations and update this file.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
 # Inverted-Index
 
 [![Build Status](https://travis-ci.org/andela-tAdedotun/Inverted-Index.svg?branch=master)](https://travis-ci.org/andela-tAdedotun/Inverted-Index)
+[![Coverage Status](https://coveralls.io/repos/github/andela-tAdedotun/Inverted-Index/badge.svg?branch=development)](https://coveralls.io/github/andela-tAdedotun/Inverted-Index?branch=development)
+[![Code Climate](https://codeclimate.com/github/andela-tAdedotun/Inverted-Index/badges/gpa.svg)](https://codeclimate.com/github/andela-tAdedotun/Inverted-Index)
```
Comment (raphaelakpan) [inline README.md]: Your `readme` file doesn't contain much information about your application

File: src/index.html [hunk 5]
```diff
@@ -9,26 +9,76 @@
...
+
+                <li value="Delete All" ng-show="moreThanOneStored">
+                  <button ng-click="deleteIndex('Delete All')" style="width: 100%; background-color: #c42940; margin-top: 20px; border-radius: 0px;">Delete All</button>
+                </li>
+              </ul>
+            </div>
+
+      </div>
+
+        <button id="littleAbout" data-toggle="collapse" data-target="#smallAbout" class="btn btn-primary">?</button>
```
Comment (raphaelakpan) [inline src/index.html]: `littleAbout` doesn't seem to be a good name choice. You can name it better and still descriptive.

Comment (taiwo-sam) [reply src/index.html]: Thanks! I'll change it.

File: README.md (hunk 4 above)
Comment (taiwo-sam) [reply README.md]: Okay, thanks. I will update it with more information.

File: .eslintrc.json (hunk 3 above)
Comment (taiwo-sam) [reply .eslintrc.json:1]: Thanks! I will do that.

**Pull request merged**